lib/Driver/ToolChains.cpp

```diff
@@ -231,6 +231,8 @@ static void addCommonFrontendArgs(const ToolChain &TC, const OutputInfo &OI,
                        options::OPT_enable_experimental_dependencies);
   inputArgs.AddLastArg(arguments,
                        options::OPT_experimental_dependency_include_intrafile);
+  inputArgs.AddLastArg(
```

Comment:
Please surround newly added code in `// SWIFT_ENABLE_TENSORFLOW` and `// SWIFT_ENABLE_TENSORFLOW END`.